upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -235,48 +283,386 @@ public function apply_addsessioncookieprefs()
-                // Recursively delete subfolders
-                self::deleteFolder($filePath);
+                if (!self::deleteFolder($filePath)) {
+                    return false;
+                }
             } else {
-                // Delete file
-                unlink($filePath);
+                if (!unlink($filePath)) {
+                    return false;
+                }
             }
```

PHPMD (2.15.0)

[error] The method deleteFolder uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)